packages/commonwealth/server/routes/createChainNode.ts

```diff
@@ -0,0 +1,51 @@
+import { AppError } from 'common-common/src/errors';
+import type { NextFunction, Request, Response } from 'express';
+import type { DB } from '../models';
+import {
+  TypedRequestBody,
+  TypedRequestQuery,
```

Comment:
There are a lot of eslint warnings in this PR that should be removed before merging